Contributing/How-to-Contribute-Land-Sector-Data-Analysis.md

```diff
@@ -0,0 +1,443 @@
+# **Moja Global Land Sector Data Analysis Guide**
+
+# **1.0 Introduction**
+
+# 1.0.0 The Land Sector Dataset
+The aim of this document is to guide contributors on how to report the analysis of [Moja Global's Land Sector Dataset](https://datasets.mojaglobal.workers.dev/0:/). The dataset is a collection of publicly available [Opensource datasets](https://github.com/moja-global/Land_Sector_Datasets) which you can chose any and work on. 
+>These datasets have been used by many countries and are effective in real-world scenarios. For each dataset information is provided on content and license. Proper permissions have been cited for all the datasets and we urge you to follow the license for every dataset before proceeding to work with it.
+
+If you have another dataset on your mind that would be beneficial to [Moja Global's mission](https://moja.global/), please feel free to reach out to us on [info@moja.global](info@moja.global) to share details about this dataset.
+
+---
+
+# 1.0.1 Data Analysis Software and Notebooks
+Concerning the data analysis software or programming language for the analysis, we encourage you to pick a softwate that supports geospatial analysis since you will be dealing with polygon files. 
+Moja Global is an open source project, the interest here is to share your processes(**code**) and results(**interpretations**) to support the open source movement. In order to efficiently doucment your analysis, we advice you use [notebooks](https://sourceforge.net/software/product/Google-Colab/alternatives). 
+
+Notebooks are documents produced by the data analysis software, which contain both computer code (e.g. python) and rich text elements (paragraph, equations, figures, links, etc…). 
+
+Notebook documents are both human-readable documents containing the analysis description and the results (figures, tables, etc..) as well as executable documents which can be run to perform data analysis. 
+With this in mind, we will be expecting different notebooks which is good since it would be at your convinience and you can reveal insight which has not been previously uncovered.
+
+---
+
+# 1.0.2 Contributing
+
+It is important to note that when you have your contribution ready, you would have to make PR to have it merged. Please refer to the [community website](https://github.com/moja-global/About_moja_global/blob/master/Contributing/How-to-Contribute-Code.md) to know how to make a pull request.

```

Comment:
Our community website lives here - https://community.moja.global/community/contributing

Reply:
Thank you @aornugent . I have made the modifications.